include/st40_api.h

```diff
@@ -84,23 +84,49 @@ enum st40_type {
 /**
  * A structure describing a st2110-40(ancillary) rfc8331 rtp header
  */
+#ifdef MTL_LITTLE_ENDIAN
 MTL_PACK(struct st40_rfc8331_rtp_hdr {
   /** Rtp rfc3550 base hdr */
   struct st_rfc3550_rtp_hdr base;
   /** Extended Sequence Number */
   uint16_t seq_number_ext;
   /** Number of octets of the ANC data RTP payload */
   uint16_t length;
-
-  struct {
-    /** the count of the total number of ANC data packets carried in the RTP payload */
-    uint32_t anc_count : 8;
-    /** signaling the field specified by the RTP timestamp in an interlaced SDI raster */
-    uint32_t f : 2;
-    /** reserved */
-    uint32_t reserved : 22;
+  union {
+    struct {
+      /** reserved */
+      uint32_t reserved : 22;
+      /** signaling the field specified by the RTP timestamp in an interlaced SDI raster
+       */
+      uint32_t f : 2;
+      /** the count of the total number of ANC data packets carried in the RTP payload */
+      uint32_t anc_count : 8;
+    } st40_rfc8331_hdr;
+    uint32_t swapped_handle_rtp_hdr;
```

Comment:
I suggest renaming to align better with existing code
```suggestion
    } first_hdr_chunk;
    uint32_t swapped_first_hdr_chunk;
```